idmega2000/visitrail
# #166172069 Initial project setup

**What does this PR do?**

It creates the initial project setup

**Description of Task to be completed?**
- [x] create basic Django project setup
- [x] create an API app
- [x] setup PostgreSQL database
- [x] add Django rest framework

**How should this be manually tested?**
- Fetch the branch `git fetch origin ch-initial-project-setup-166172069`
- Checkout into the branch `git ch-initial-project-setup-166172069`
- Run `python manage.py migrate` to run migrations
-  Run `python manage.py runserver to start the app`
- The app should be started on port 8000

**Any background context you want to provide?**
This is the beginning of the visitrail project

**What are the relevant pivotal tracker stories?**
[#166172069](https://www.pivotaltracker.com/story/show/166172069)

**Screenshots (if appropriate)**

N/A
